doc/cephfs/snap-schedule.rst

```diff
@@ -38,6 +38,13 @@ below). By default
 the start time is last midnight. So when a snapshot schedule with repeat
 interval `1h` is added at 13:50
 with the default start time, the first snapshot will be taken at 14:00.
+The time zone is assumed to be UTC if none is explicitly included in the string.
+An explicit time zone will be mapped to UTC at execution.
+The start time must be in ISO8601 format. Examples below:
+
+UTC: 2022-08-08T05:30:00 i.e. 5:30 AM UTC, without explicit time zone offset
+UTC: 2022-08-08T09:00:00+00:00 i.e. 9:00 AM UTC
```

Comment:
Can we add some other time zone example here instead of UTC as the first one already takes care of it, maybe PST or CST?

Reply:
added Tel Aviv time, IDT

Reply:
Looks good